apps/dashboard/src/components/editor/index.js

```diff
@@ -23,13 +22,9 @@ import { editorSettings } from './settings';
 import Toolbar from './toolbar';
 import UnpublishedChangesNotice from './unpublished-changes-notice';
 import { useAutosave } from './use-autosave';
+import { useStylesheet } from '@crowdsignal/hooks';
 
-/**
- * Style dependencies
- */
-import './style.scss';
-
-const Editor = ( { projectId } ) => {
+const Editor = ( { projectId, theme = 'leven' } ) => {
```

Comment:
Nit: while it doesn't hurt, we can leave the theme hardcoded for now. Reason being, when we eventually implement theme support on the backend, this will become a property on `project` itself.

Reply:
Yeah, the idea here and in the `project-renderer` was to keep the `leven` theme as default, since it's our default theme at the moment, but also provide a way to visualize other themes if you pass the `theme` argument on the URL (`?theme=crowdsignal`).